testsuite/gnat2goto/tests/ukni_loop_parameter_specification/test.adb

```diff
@@ -0,0 +1,16 @@
+procedure Test is
+
+   type Unsigned_8 is mod 2 ** 8;
+   for Unsigned_8'Size use 8;
+
+   type Unsigned_16 is mod 2 ** 16;
+   for Unsigned_16'Size use 16;
+
+   RAM_Start : constant Unsigned_16 := 16#0200#;
+   RAM_End   : constant Unsigned_16 := 16#2199#;
+
+begin
+   for I in Unsigned_16 range RAM_Start .. RAM_End loop
+      null;
```

Comment:
Just so that CBMC gets exercised a bit more, is it worth adding some additional assertions into the test case, and checking that CBMC gives the expected results? e.g. asserting something about I inside the loop perhaps?

Reply:
Agreed - I have added a pragma Asesrt that `I < 16#2200#`.